src/ModelObjects/queries.rs

```diff
@@ -28,7 +28,7 @@ where
         return Ok(None);
     }
 
-    let queries = parse_queries::parse_to_expression_tree(&s);
+    let queries = parse_queries::parse_to_expression_tree(&s).unwrap();
```

Comment:
Error?

Reply:
@t-lohse this is a problem for the "cause of failure" group. Sebastian said we could just leave them be for now.